### PR TITLE
Output checking docs structure

### PR DESCRIPTION
**Pull request by LFISHER7**
The docs around output checking have grown in size. In preparation for reviewing the content, this splits the page into more manageable chunks.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ nav:
   - About OpenSAFELY:
       - Our philosophy: open-methods.md
       - Security: security-levels.md
+      - Five Safes: five-safes.md
```
Comment (milanwiedemann) [inline mkdocs.yml:13]:
```suggestion
      - Safe research access to data: five-safes.md
```

File: mkdocs.yml [hunk 2]
```diff
@@ -40,7 +41,12 @@ nav:
           - Reusable actions: actions-reusable.md
       - Jobs site: jobs-site.md
       - Level 4 server: level-4-server.md
-      - Safe Outputs and requesting release of files from the Level 4 server: releasing-files.md
+      - Requesting release of files from the Level 4 server:
+        - Overview: releasing-files-intro.md
+        - Applying statistical disclosure controls to your research outputs: sdc.md
+        - Requesting release of research outputs: requesting-file-release.md
+        - Review of outputs requested for release: output-checking.md
+        - File release: releasing-files.md
```
Comment (milanwiedemann) [inline mkdocs.yml:49]: Some ideas to improve (I tried to shorten some, and use _clearer_ terms) and  the headings and subheadings. Ignore if you're happy with your titles

```suggestion
      - Releasing research outputs:
        - Overview: releasing-files-intro.md
        - Applying statistical disclosure controls: sdc.md
        - Requesting release of research outputs: requesting-file-release.md
        - Review process of output requests: output-checking.md
        - Release of approved outputs: releasing-files.md
```

Comment (milanwiedemann) [reply mkdocs.yml:49]: In case you decide to make any changes to subheadings, probably also change the title of the page

**Review by milanwiedemann — APPROVED** [review]
Looks good :) I had some ideas how to _improve_ the subheadings, but ignore my comments if you're happy with the current headings.

File: mkdocs.yml (hunk 1 above)
Comment (LFISHER7) [reply mkdocs.yml:13]: Good idea. I've updated this to `Safe data access: The Five Safes`, so it's best of both

File: mkdocs.yml (hunk 2 above)
Comment (LFISHER7) [reply mkdocs.yml:49]: I've left reference to Level 4 files in the main title - I think this is important, but have updated the subsection titles in line with your suggestions

**Pull request merged**